index.html

```diff
@@ -4381,6 +4377,75 @@ <h3>Parsing content</h3>
       [[RFC2048]] and [[RFC2046]].
     </p>
   </section>
+  <section> <h3>Things that users should be made aware of when using NFC</h3>
+    <p>
+      This section details some of the things that users ought to be aware
+      of when using NFC. It is recommendated that implementations help educate
+      the users of given facts before or when related NFC actions are performed. 
+    </p>
+    <h4>Data that is read is shared with site</h4>
+    <p>
+      When a site has access to read NFC content, then the data of the scanned
+      tags is shared with the site, in a similar way to uploading files and images.
+      The user has to trust that the site handles this data properly and in the
+      intended manner.
```

Comment:
Not sure if the "user has to trust" is proper language for security :)

Reply:
will have to? :)

Reply:
The trust relationship between the user and the site needs to be established independently of using this API.
(complex, but expresses the point)

Reply:
Sounds very cryptic to me. @anssiko

Reply:
IoW, the trust between the user and the site is separate from using Web NFC. 
First, the user needs to trust the site (including data handling), then allows using NFC (including NFC data handling).